test/tests/variables/optionally_coupled/tests

```diff
@@ -17,4 +17,44 @@
     input = 'optionally_coupled_system.i'
     exodiff = 'optionally_coupled_system_out.e'
   [../]
+
+  [./two_coupled_default_1]
+    type = 'Exodiff'
+    input = 'optionally_coupled_twovar.i'
+    exodiff = 'two_coupled_default_1.e'
+    cli_args = 'Outputs/file_base=two_coupled_default_1'
+    requirement = "Coupleable shall provide an interface that allows providing default values for a vector of coupled variables"
+    design = 'Coupleable.md'
+    issues = '#11920'
+  [../]
+
+  [./two_coupled_default_2]
+    type = 'Exodiff'
+    input = 'optionally_coupled_twovar.i'
+    exodiff = 'two_coupled_default_2.e'
+    cli_args = 'Kernels/optional_coupling/v="3 5" Outputs/file_base=two_coupled_default_2'
+    requirement = "Coupleable shall provide an interface that allows providing default values for a vector of coupled variables"
+    design = 'Coupleable.md'
+    issues = '#11920'
+  [../]
+
+  [./two_coupled_default_fail_few_args]
+    type = 'RunException'
+    input = 'optionally_coupled_twovar.i'
+    cli_args = 'Kernels/optional_coupling/v="3"'
+    expect_err = 'Should always have two coupled components.'
+    requirement = "Coupleable shall provide an interface that allows providing default values for a vector of coupled variables"
+    design = 'Coupleable.md'
+    issues = '#11920'
+  [../]
+
+  [./two_coupled_default_fail_mixed_input]
+    type = 'RunException'
+    input = 'optionally_coupled_twovar.i'
+    cli_args = 'Kernels/optional_coupling/v="3 v"'
+    expect_err = 'MOOSE does not currently support a coupled vector where some parameters are reals and others are variables'
+    requirement = "Coupleable shall provide an interface that allows providing default values for a vector of coupled variables"
```

Comment:
The requirement you are testing is actually: "The system shall report an error when users couple constant values and variables together." That's why `_have_coupled_default` is `bool` and not `std::vector<bool>`, right?